global.json

```diff
@@ -1,6 +1,6 @@
 {
   "sdk": {
-    "version": "3.0.100-preview4-010309"
+    "version": "3.0.100-preview4-011136"
```

Comment:
@nkolev92 this is the version of the SDK we're trying to update to.

Reply:
![image](https://user-images.githubusercontent.com/2696087/55762654-9a8a9880-5a18-11e9-9abb-82d982451295.png)
https://github.com/dotnet/toolset/pull/593#issuecomment-479250753

@nguerrera We're seeing this feature show up in P4 builds of the SDK...did something changed in the last 6 days since this comment, or did this unintentionally get merged into P4?

Reply:
How does the feature being present break things?

I intentionally did not revert it because I figured we should get coverage of latest bits. There was no expectation of anything breaking. I chatted briefly with @nkolev92 after that comment.

If necessary, we can revert the nuget version in P4.

Reply:
Note that @pranavkm updated to the latest SDK available as of today.

Reply:
Understood. The question is why is taking a nuget with the frameworkReferences feature a breaking change?

Reply:
It breaks the aspnet build because we have some custom MSBuild tasks reading the nuspec, and NuGet code throws when unrecognized elements appear in the metadata. 

FWIW, though, the reason we have these custom MSBuild tasks was to workaround the absence of this NuGet feature...so having access to this SDK unblocks us to start using `<FrameworkReference>` the *right* way.

@pranavkm was there any particular reason you were trying to update to the latest SDK? If not, maybe we should combine this update with a solution for #4257

Reply:
Right, essentially there's a mismatch in the toolset which uses nuget 5.1.0-rtm.5921 to build our packages and the NuGet.Build.Tasks that we use to post process the packages built. To reduce the work that's required to update the SDK which to my understanding is blocking us, why don't we just update the version of NuGet.Build.Tasks that we use to 5.1.0-rtm.5921 as well?

Reply:
Should be available at https://dotnet.myget.org/feed/nuget-build/package/nuget/NuGet.Build.Tasks/5.1.0-rtm.5921+27af96bdb7ba8d6d7ea9ad53fc76cd1d1aa80703.

Reply:
> @pranavkm was there any particular reason you were trying to update to the latest SDK? If not, maybe we should combine this update with a solution for #4257

There are changes in the WebSDK that are require to the test the preview4 templates.